domain/include/cstone/gravity/multipole.hpp

```diff
@@ -221,7 +221,9 @@ util::tuple<T1, T1, T1, T1> particle2particle(T1 tx,
     T1 azLoc = 0;
     T1 uLoc  = 0;
 
-    #pragma clang loop vectorize(enable)
+	#if defined(__llvm__) || defined(__clang__)
+    	#pragma clang loop vectorize(enable)
+	#endif
```

Comment:
wrong alignment, spurious tab-characters?

Reply:
You were right. I reinstalled the IDE eclipse some days ago. I forgot to change in the 'Formatted' tab by spaces. Now It is fix and commit.